src/exportStrategies.js

```diff
@@ -113,10 +124,10 @@ export const CSVStream = async ({
       }
 
       // Format the values in the current chunk with the passed in formatRules and fill any blank props
-      let formattedData = formatValues(formatRules, includeKeys)(chunk)
+      let [formattedData, fullFormattedData] = formatValuesWithOmits(formatRules, includeKeys, omitFieldsFromResult)(chunk)
 
       // Convert data to CSV rows
-      let rows = extractValues(formattedData, includeKeys)
+      let rows = extractValues(formattedData, _.difference(includeKeys, omitFieldsFromResult))
```

Comment:
Do we really need `formatValuesWithOmits`? It looks like as long as we pass  _.difference(includeKeys, omitFieldsFromResult) as `keys` to `extractValues` then the resulting csv rows will still skip the fields we don't want.

Reply:
that may work, let me try it out...

Reply:
yep, that works -- updated. ty, @dshishkov !